packages/ui-components/src/components/tooltip/tooltip.tsx

```diff
@@ -55,6 +58,19 @@ export class KvTooltip implements ITooltip {
 	};
 
 	private showTooltipHandler = () => {
+		if (this.truncate) {
+			const collapsedElement = getCollapsedElement(this.el.childNodes);
```

Comment:
Maybe it would be a nice thing to move this logic to a helper. Something like, `isElementCollpased(element: HTMLElement): boolean`?